src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json

```diff
@@ -35,6 +35,30 @@
             "linkType": "kfs"
           },
           {
+            "label": "Disbursement Voucher",
+            "link": "financialDisbursementVoucher.do?methodToCall=docHandler&command=initiate&docTypeName=DV",
+            "linkType": "kfs",
+            "permission": {
+              "templateNamespace": "KR-SYS",
+              "templateName": "Initiate Document",
+              "details": {
+                "documentTypeName": "DV"
+              }
+            }
+          },
+          {
+            "label": "Recurring Disbursement Voucher",
```

Comment:
I asked Cathy about the order, her response: "together..  put the DV in alphabetic order, have the recurring DV follow."